src/components/Map/GeoJSONTileLayer.js

```diff
@@ -160,6 +183,7 @@ class GeoJSONTileLayer extends TileLayer {
       x: coords.x,
       y: coords.y,
       z: this._getZoomForUrl(),
+      c: this._cacheBuster,
```

Comment:
You've probably put more thoughts into this so I might have overlooked something… from what I see this shouldn't be necessary - the cache headers for this should be set correctly on the server side so the browser doesn't store tiles for too long, and fastly caches are purged when something changes. Adding this parameter would add responses to the varnish storage that are not reusable by anybody as the tiles have a long TTL there, and reduce the hit rate. Isn't there an alternative for this, e.g. a parameter we can set in the request options so the browser cache is not used for sure? How about setting a cache buster boolean flag that enables this behavior and is reset directly after?

Reply:
My browser was aggressively caching equipment map tiles, sadly. Either some of the fastly logic is wrong, or leaflet is doing something weird, or something else is misconfigured. I was still getting the same grey tiles without the cache buster :/

Reply:
Thanks for debugging + info! I'll take over from here and try to fix this.